### PR TITLE
Db fixes prod 2 7

### PR DESCRIPTION
**Pull request by arnaud-morvan**
TODO: backport 2.7-advance and 2.8

### CODE REVIEW

File: docker-compose-lib.yaml [hunk 1]
```diff
@@ -29,6 +29,9 @@ services:
       - SQLALCHEMY_POOL_RECYCLE
       - SQLALCHEMY_POOL_SIZE
       - SQLALCHEMY_MAX_OVERFLOW
+      - SQLALCHEMY_SLAVE_POOL_RECYCLE
+      - SQLALCHEMY_SLAVE_POOL_SIZE
+      - SQLALCHEMY_SLAVE_MAX_OVERFLOW
```
Comment (sbrunner) [inline docker-compose-lib.yaml]: This should be done in c2cgeoportal :-)

File: docker-compose-lib.yaml [hunk 2]
```diff
@@ -218,6 +221,9 @@ services:
...
```
Comment (sbrunner) [inline docker-compose-lib.yaml]: Same

**Review by sbrunner — APPROVED** [review]

File: docker-compose-lib.yaml (hunk 2 above)
Comment (arnaud-morvan) [reply docker-compose-lib.yaml]: @sbrunner : Yes we have default values in Dockerfile, but I would like to set them in docker-compose too, using env files.

We may have different values in demo and production projects.

File: docker-compose-lib.yaml (hunk 1 above)
Comment (arnaud-morvan) [reply docker-compose-lib.yaml]: Done: https://github.com/camptocamp/c2cgeoportal/pull/9658

**Comment by arnaud-morvan** [comment]
I created a PR in c2cgeoportal : https://github.com/camptocamp/c2cgeoportal/pull/9658
I suppose we can merge this without creating any conflicts for upgrade

**Comment by arnaud-morvan** [comment]
Note that we already have reduced values in env.project : https://github.com/camptocamp/demo_geomapfish/blob/prod-2-7/env.project#L17:L18

**Pull request merged**